scripts/components/security/build.sh

```diff
@@ -59,22 +60,16 @@ if [ -z "$VERSION" ]; then
     exit 1
 fi
 
-OPENSEARCH_RELEASE=$VERSION
 [[ "$SNAPSHOT" == "true" ]] && VERSION=$VERSION-SNAPSHOT
 [ -z "$OUTPUT" ] && OUTPUT=artifacts
 
-# see https://github.com/opensearch-project/security/pull/1409
-PLUGIN_VERSION=$(cat plugin-descriptor.properties | grep ^version= | cut -d= -f2 | sed "s/-SNAPSHOT//")
-[[ "$SNAPSHOT" == "true" ]] && PLUGIN_VERSION=$PLUGIN_VERSION-SNAPSHOT
+mkdir -p $OUTPUT
 
-sed -i -e "s/\(^opensearch\.version=\).*\$/\1${OPENSEARCH_RELEASE}/" plugin-descriptor.properties
-sed -i -e "s/\(^version=\).*\$/\1${PLUGIN_VERSION}/" plugin-descriptor.properties
-sed -i -e "s/\(<opensearch.version>\).*\(<\/opensearch.version>\)/\1${VERSION}\2/g" pom.xml
-sed -i -e "1,/<version>/s/\(<version>\).*\(<\/version>\)/\1${PLUGIN_VERSION}\2/g" pom.xml
+./gradlew assemble --no-daemon --refresh-dependencies -DskipTests=true -Dopensearch.version=$VERSION -Dopensearch_version=$VERSION -Dbuild.snapshot=$SNAPSHOT
```

Comment:
Note, there is a typo in the security build.gradle adding, so I am adding the additional version name until the fix can be made / merged into security.  This duplicate will get cleaned up out of band so the build isn't block while we wait.

https://github.com/opensearch-project/security/blob/5b994af53f8f5a9b258b2c66f844ce1ac88d6c1e/build.gradle#L60